forge-game/src/main/java/forge/game/ability/SpellAbilityEffect.java

```diff
@@ -82,8 +82,7 @@ public final String getStackDescriptionWithSubs(final Map<String, String> params
                 if (params.containsKey("SpellDescription")) {
                     if (rawSDesc.contains(",,,,,,")) rawSDesc = rawSDesc.replaceAll(",,,,,,", " ");
                     if (rawSDesc.contains(",,,")) rawSDesc = rawSDesc.replaceAll(",,,", " ");
-                    String spellDesc = CardTranslation.translateSingleDescriptionText(rawSDesc,
-                            sa.getHostCard().getName());
+                    String spellDesc = CardTranslation.translateSingleDescriptionText(rawSDesc, sa.getHostCard());
```

Comment:
this doesn't feel right:
`Card.getName()` has additional logic which you now seem to bypass for the plain CardState value? :dizzy_face:

Reply:
oh nevermind, I think I misread that part

Reply:
I saw that `Card.getName()` has logic for applying changed names, but I ended up getting the name to use for translations - the "translation key" as I've been calling it - from the card state, which I believe does step around the `ChangedCardName` logic. I'm not sure if that's the right call. The thing that seemed weird about using `Card.getName()` is that changing the name would change the oracle text that the translator supplies. Is that intended? If not I could rework it.